plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties

```diff
@@ -11,3 +11,6 @@ maven.plugin.marker=Obsolete Maven App Engine Plugin
 undefined.servlet.marker=Unresolved Servlet Class
 undefined.servlet.mapping=Undefined Servlet Mapping
 unresolved.jsp.file=Unresolved JSP File
+remove.application.element=Remove application element
```

Comment:
This should probably be a description, not a verb. E.g. "Ignored application element"

Reply:
Agree